cmd/ptp4u/main.go

```diff
@@ -44,8 +44,9 @@ func main() {
 	flag.DurationVar(&c.MinSubInterval, "minsubinterval", 1*time.Second, "Minimum interval of the sync/announce subscription messages")
 	flag.DurationVar(&c.MaxSubDuration, "maxsubduration", 1*time.Hour, "Maximum sync/announce/delay_resp subscription duration")
 	flag.StringVar(&c.TimestampType, "timestamptype", timestamp.HWTIMESTAMP, fmt.Sprintf("Timestamp type. Can be: %s, %s", timestamp.HWTIMESTAMP, timestamp.SWTIMESTAMP))
-	flag.DurationVar(&c.UTCOffset, "utcoffset", 37*time.Second, "Set the number of workers. Ignored if shm is set")
-	flag.BoolVar(&c.SHM, "shm", false, "Use Share Memory Segment to determine UTC offset periodically")
+	flag.DurationVar(&c.UTCOffset, "utcoffset", 37*time.Second, "Set the number of workers. Ignored if shm or leapsectz are set")
```

Comment:
"Set the number of workers..." wat? Maybe "Set the offset from UTC"?

Reply:
![image](https://user-images.githubusercontent.com/4749052/152255113-ba280ee9-b367-4e3e-83a4-f408f2e05588.png)